core/templates/pages/topics-and-skills-dashboard-page/modals/create-new-skill-modal.component.spec.ts

```diff
@@ -47,13 +46,13 @@ describe('Create new skill modal', () => {
         FormsModule
       ],
       declarations: [
-        CreateNewSkillModalComponent,
-        SchemaBasedEditorDirective
+        CreateNewSkillModalComponent
       ],
       providers: [
         NgbActiveModal,
         ChangeDetectorRef,
-      ]
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
```

Comment:
Why add this? I still worry about errors like the one mentioned [here](https://medium.com/@fivedicephoto/why-you-shouldnt-use-no-errors-schema-in-angular-unit-tests-cdd478c30782).

Reply:
We pretty much need it for unit-tests.